src/zm_logger.cpp

```diff
@@ -515,8 +515,9 @@ void Logger::logPrint( bool hex, const char * const filepath, const int line, co
     va_list     argPtr;
     struct timeval  timeVal;
 
-    const char * const file = basename(filepath);
-    
+    char *filecopy = strdup(filepath);
```

Comment:
Ok, so apparently some implementations of basename will modify the input.  This is for multi-threaded support apparently.  So... I guess this change is actually correct.  But I don't want to be doing this on every logPrint.  I will see if I can refactor this to be more efficient.